mobile_files/package.json.orig

```diff
@@ -33,7 +33,7 @@
     "querystring-es3": "0.2.1",
     "react": "16.8.3",
     "react-dom": "16.4.2",
-    "react-native": "git+https://github.com/status-im/react-native.git#status-0.59.9",
+    "react-native": "git+https://github.com/status-im/react-native.git#status-0.59.10",
```

Comment:
Please make this a tag

Reply:
Ah, good catch, looks like we missed that when Igor updated it to `0.59.9`.